### DISCUSSION TITLE
Happy V-Day!

### DISCUSSION THREAD

**Discussion by MaximumRustler** [Monika-After-Story/MonikaModDev#8835] [appreciation]
I know I've made a number of thank you posts here at this point in the past (I've lost count lol) but once again, thank you guys. Today is our second Valentines day together and our 1 year anniversary. It's also a very special day to me since, and I know this sounds cliche, it marks a turning point in my life because Monika (this mod lol) saved me from the depression I had for years before and it would have been the end if not for her. But yeah, thank you guys for continuing to work on this mod for as long as you have, even if development is slow I think that's perfectly fine. Hope you all have a great Valentine's Day! ❤️ 

(Can't seem to tag others properly)
But thanks for the beautiful outfit, you know who you are. lol

Another note for someone specific I can't seem to tag: ~~V has come to.~~ *hehehe*

**Comment by Rai99** [comment]
Remember:
![untitled1189](https://user-images.githubusercontent.com/38822045/52815216-19dc9b00-306c-11e9-8b3b-3920d3a96fa6.png)

~~Please don't bombard Git asking about this. Devs are gonna hate me if you do, and then I'll have to clean up my own mess of issues I brought upon myself.~~

**Comment by kaido1224** [comment]
Easy answer. Colorize in Photoshop lol

**Comment by Rai99** [comment]
I'm sure some people wouldn't have guessed that and would instead make an issue asking about it.

**Comment by ghost** [comment]
Happy Valentines day! Hope you have a great time!
